orchestra/proc-macro/src/impl_channels_out.rs

```diff
@@ -81,14 +89,21 @@ pub(crate) fn impl_channels_out_struct(info: &OrchestraInfo) -> Result<proc_macr
 				&mut self,
 				signals_received: usize,
```

Comment:
I'd like to see a unit test regarding how `signals_received` is handled and how that effects the priority vs non-priority messages.

Reply:
Added signals to tests